src/main/scala/it/pagopa/interop/backendforfrontend/service/types/CatalogProcessServiceTypes.scala

```diff
@@ -96,7 +96,15 @@ object CatalogProcessServiceTypes {
         name = es.name,
         description = es.description,
         technology = es.technology.toProcess,
-        mode = es.mode.toProcess
+        mode = es.mode.toProcess,
+        descriptor = CatalogProcess.DescriptorSeedForEServiceCreation(
+          description = None,
+          audience = Seq(),
```

Comment:
constraint in process: min length 1

Reply:
Same as https://github.com/pagopa/interop-be-backend-for-frontend/pull/302#discussion_r1679156646

Reply:
Update: the constraint is being edited (https://github.com/pagopa/interop-be-monorepo/pull/762)